server/db/models/User.ts

```diff
@@ -0,0 +1,23 @@
+import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
+
+@Entity()
+export class User {
+  @PrimaryGeneratedColumn()
+  id: number | undefined;
```

Comment:
Hapa pia we will not have a situation where the fields are `undefined` so uneza toa the undefined type on the fields.